src/actions/onboardingActions.js

```diff
@@ -276,19 +282,12 @@ export const validateUserDetailsAction = ({ username }: Object) => {
       type: UPDATE_WALLET_STATE,
       payload: CHECKING_USERNAME,
     });
-    const { mnemonic, importedWallet } = currentState.wallet.onboarding;
+    const { mnemonic } = currentState.wallet.onboarding;
     const mnemonicPhrase = generateMnemonicPhrase(mnemonic.original);
     dispatch(generateWalletMnemonicAction(mnemonicPhrase));
     await delay(200);
 
-    let wallet = importedWallet;
```

Comment:
Became unused.

Reply:
This will conflict another PR's handling of preConnectionKey generation call https://github.com/pillarwallet/pillarwallet/pull/1028/commits/a5a48269ae63e0d37146d8120ac720124a5ccb69

Reply:
I think it won't, although the code in your PR only needs `importedWallet` which is still there at the point you need it, is this ok?